lms/djangoapps/verify_student/views.py

```diff
@@ -504,6 +520,8 @@ def _redirect_if_necessary(
 
         # Redirect if necessary, otherwise implicitly return None
         if url is not None:
+            if waffle.switch_is_active('add-utm-params'):
```

Comment:
@mikedikan I would add it manually. I'm including this as a temporary safeguard in case this is not what we want. I would remove the switch and add a unit test if this code is correct (wouldn't want to add a migration to add and migration to remove it)